src/ray/gcs/gcs_server/gcs_server_main.cc

```diff
@@ -27,6 +29,7 @@ DEFINE_string(redis_address, "", "The ip address of redis.");
 DEFINE_bool(redis_enable_ssl, false, "Use tls/ssl in redis connection.");
 DEFINE_int32(redis_port, -1, "The port of redis.");
 DEFINE_string(log_dir, "", "The path of the dir where log files are created.");
+DEFINE_int64(log_rotation_max_size, 0, "Max size to rotate logs for GCS.");
```

Comment:
We already have `RAY_ROTATION_MAX_BYTES`, I don't think we need to introduce a new env var

Reply:
Make sense